.github/workflows/tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: "Tests"
+
+on: [push, pull_request]
+
+jobs:
+  phpunit:
+    name: "Tests"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.3"
+          - "7.4"
+          - "8.0"
+          - "8.1"
+          - "8.2"
+        deps:
+          - "lowest"
+          - "highest"
+        coverage:
+          - "false"
+        include:
+          - deps: "highest"
+            php-version: "8.2"
```

Comment:
Are these declarations desired? Which are their purposes?

Reply:
Yes it's my wish. I want the job with the code coverage to be launched only with the latest version of PHP, for other versions I think it's not a useful information.

Reply:
So, being already present in the matrixes, turns out to be redundant, we could therefore remove `deps` and `php-version` under the `include` section.

Reply:
I didn't notice the introduction of the `coverage` property.